twitter/tests/test_tweepy_scraper.py

```diff
@@ -29,11 +30,42 @@ def test_add_user(self):
         self.assertIsNone(TwitterUser.query.filter_by(user_name='@TestUser').first())
 
         tweepy_scraper.add_user(data)
-
         test_user = TwitterUser.query.filter_by(user_name="@TestUser").first()
+
         self.assertEqual(test_user.user_name, data['user_name'])
         self.assertEqual(test_user.location, data['location'])
 
+    def test_parse_status(self):
+        self.maxDiff = None
+        status = Mock(
+            user_name='@fake_user_name',
+            user_location='fake_location',
+            coordinates='fake_coordinates',
+            text='fake_test',
+            # symbols_in_text='fake_symbols_in_fake_text',
+            # hashtags='fake_hashtags',
+            created_at='fake_created_date',
+            #lead_score=3,
```

Comment:
I changed all of these to have no spaces around the equals sign. When assigning object parameters or function keyword arguments, you don't leave spaces around the equals sign. https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements